src/Calculator/Views/GraphingCalculator/GraphingNumPad.xaml.cpp

```diff
@@ -104,3 +193,55 @@ void GraphingNumPad::SetOperatorRowVisibility()
     Row1->Visibility = rowVis;
     InvRow1->Visibility = invRowVis;
 }
+
+void GraphingNumPad::Button_Clicked(Platform::Object ^ sender, Windows::UI::Xaml::DependencyPropertyChangedEventArgs ^ /*e*/)
+{
+    auto mathRichEdit = GetActiveRichEdit();
+    auto button = dynamic_cast<CalculatorApp::Controls::CalculatorButton ^>(sender);
+    if (mathRichEdit != nullptr && sender != nullptr)
+    {
+        auto id = button->ButtonId;
+        auto output = buttonOutput.find(id);
+        mathRichEdit->InsertText(std::get<0>(output->second), std::get<1>(output->second), std::get<2>(output->second));
+    }
+}
+
+void GraphingNumPad::SubmitButton_Clicked(Platform::Object ^ sender, Windows::UI::Xaml::RoutedEventArgs ^ e)
+{
+    auto mathRichEdit = GetActiveRichEdit();
+    if (mathRichEdit != nullptr)
+    {
+        mathRichEdit->SubmitEquation();
+    }
+}
+
+void GraphingNumPad::ClearButton_Clicked(Platform::Object ^ sender, Windows::UI::Xaml::RoutedEventArgs ^ e)
+{
+    auto mathRichEdit = GetActiveRichEdit();
```

Comment:
I still see this button clearing the textbox and giving the textbox below focus, I think focus should stay with the cleared textbox.

Reply:
I could have sworn I'd tested that, but apparently not.  looking into fixing that.